api/models/reviews.models.js

```
@@ -25,3 +25,38 @@ exports.fetchAllReviews = () => {
     )
     .then((res) => res.rows);
 };
+
+exports.setReviewVotes = async (body, params) => {
+  const { inc_votes } = body;
+  const { review_id } = params;
+  if (isNaN(inc_votes) || isNaN(review_id)) {
+    return Promise.reject({ status: 400, msg: "Invalid format" });
+  }
+
+  const checkIDExists = await db.query(
+    `
+    SELECT * FROM reviews
+    WHERE review_id = $1;
+    `,
+    [review_id]
+  );
+
+  if (checkIDExists.rowCount === 0) {
+    return Promise.reject({
+      status: 404,
+      msg: "ID not found",
+    });
+  }
```

PSQL should throw an Error if the article ID does not exist saving you having to do another database query.